<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="THB-ARG-455.local" ignored="0" name="Test" tests="5" failures="2" timestamp="2022-11-27T23:15:24 EET" time="10.511" errors="0">
  <testcase name="runScenario" time="2.694" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.AssertionError" message="expected [404] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [404] but found [200]
at stepDefinations.TestOrder.verify_if_status_code_is_equal(TestOrder.java:39)
at ✽.Verify if status code is equal 404(file:///Users/maltunbas/git/APIProjects/APITemplateProject-PetStore/src/test/java/features/petOrderValidations.feature:7)
... Removed 6 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase name="runScenario" time="1.836" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.AssertionError" message="expected [404] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [404] but found [200]
at stepDefinations.TestOrder.verify_if_status_code_is_equal(TestOrder.java:39)
at ✽.Verify if status code is equal 404(file:///Users/maltunbas/git/APIProjects/APITemplateProject-PetStore/src/test/java/features/petOrderValidations.feature:7)
... Removed 6 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase name="runScenario" time="2.091" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="1.65" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="1.819" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
</testsuite> <!-- Test -->
